includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -162,6 +165,118 @@ public static function get_update_data() {
 		return self::_update_check( $payload );
 	}
 
+	/**
+	 * Get translations updates informations.
+	 *
+	 * Scans through all subscriptions for the connected user, as well
+	 * as all Woo extensions without a subscription, and obtains update
+	 * data for each product.
+	 *
+	 * @return array Update data {product_id => data}
+	 */
+	public static function get_translations_update_data() {
+		$payload = array();
+
+		$installed_translations = wp_get_installed_translations( 'plugins' );
+
+		$locales = array_values( get_available_languages() );
+		/**
+		 * Filters the locales requested for plugin translations.
+		 *
+		 * @since 3.7.0
+		 * @since 4.5.0 The default value of the `$locales` parameter changed to include all locales.
+		 *
+		 * @param array $locales Plugin locales. Default is all available locales of the site.
+		 */
+		$locales = apply_filters( 'plugins_update_check_locales', $locales );
+		$locales = array_unique( $locales );
+
+		// Scan local plugins which may or may not have a subscription.
+		$plugins                 = WC_Helper::get_local_woo_plugins();
+		$active_woo_plugins      = array_intersect( array_keys( $plugins ), get_option( 'active_plugins', array() ) );
+
+		/*
+		* Use only plugins that are subscribed to the automatic translations updates.
+		*/
+		$active_for_translations = array_filter(
+			$active_woo_plugins,
+			function( $plugin ) use ( $plugins ) {
+				return apply_filters( 'woocommerce_translations_updates_for_' . $plugins[ $plugin ]['slug'], false );
+			}
+		);
+
+		if ( wp_doing_cron() ) {
+			$timeout = 30;
+		} else {
+			// Three seconds, plus one extra second for every 10 plugins.
+			$timeout = 3 + (int) ( count( $active_for_translations ) / 10 );
+		}
+
+		$request_body = array(
+			'locales' => $locales,
+			'plugins' => array(),
+		);
+
+		foreach ( $active_for_translations as $active_plugin ) {
+			$plugin = $plugins[ $active_plugin ];
+			$request_body['plugins'][ $plugin['slug'] ] = array( 'version' => $plugin['Version'] );
+		}
+
+		$raw_response = wp_remote_post(
+			'https://translate.wordpress.com/api/translations_updates/woocommerce',
+			array(
+				'body'        => json_encode( $request_body ),
+				'headers'     => array( 'Content-Type: application/json' ),
+				'timeout'     => $timeout,
+			)
+		);
+
+		// Something wrong happened on the translate server side.
+		$response_code = wp_remote_retrieve_response_code( $raw_response );
+		if ( 200 !== $response_code ) {
+			return array();
+		}
+
+		// General error.
+		if ( is_wp_error( $raw_response ) ) {
```

Comment:
`wp_remote_retrieve_response_code()` returns `''` when the parameter is a `WP_Error` so this check shouldn't be necessary.

Reply:
Cool. I have missed that.